src/node_sea.cc

```diff
@@ -371,6 +400,18 @@ std::optional<SeaConfig> ParseSingleExecutableConfig(
     result.flags |= SeaFlags::kUseCodeCache;
   }
 
+  std::optional<std::unordered_map<std::string, std::string>> assets_opt =
+      parser.GetTopLevelDictOfStrings("assets");
+  if (!assets_opt.has_value()) {
+    FPrintF(stderr,
+            "\"assets\" field of %s is not a map of strings\n",
+            config_path);
+    return std::nullopt;
+  } else if (!assets_opt.value().empty()) {
```

Comment:
```suggestion
  if (assets_opt.has_value() && !assets_opt.value().empty()) {
```

Missing *asset* key in *sea-config.json* should be allowed.